src/main.ts

```diff
@@ -58,6 +58,11 @@ export async function run() {
       auth.configAuthentication(registryUrl, alwaysAuth);
     }
 
+    const enableCorepack = core.getBooleanInput('corepack');
+    if (enableCorepack) {
+      await exec.exec('corepack', ['enable']);
```

Comment:
Note this will work with pnpm and yarn, but it wont work with npm which requires `corepack enable npm` explicitly.

Reply:
One option is to change the configuration from `corepack: <boolean>` to `corpack: string[]` so that the user can pass an array of args. Then it could look like `corepack: ['enable', 'npm']`

The other solution is to do the same thing that PR #482 is doing and read `packageManager` config to set the correct value.

Reply:
IMO the best solution would be to accept `true | <package manager>` and add the value to the args if it isn't `true`